doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,634 @@
+Since the initial particle layout plays an important role in the regime of the pneumatic conveying, the particles are already forming a plug at the beginning of the CFD-DEM simulation.
+Three simulation runs need to be performed.
+First, we use ``lethe-particles`` to load the particles with the file ``loading-particles.prm``.
+The setup of the simulation is a pipe with a solid objects (mesh) that represents the shape of a slug.
+The insertion of particles are done with the plane insertion.
+Second, we use ``lethe-particles`` to settle the particles with the file ``settling-particles.prm``. Only the direction of gravity is changed.
+Finally, we use ``lethe-fluid-particles`` to simulate the dense pneumatic conveying with the file ``pneumatic-conveying.prm`` with periodic boundary conditions.
+We enable checkpointing in order to write the DEM checkpoint files which will be used as the starting point of the CFD-DEM simulation.
+The geometry of the pipe and the particle properties are based on the work of Lavrinec *et al*. [#lavrinec2021]_
+
+
+---------------------------
+Running the Simulations
+---------------------------
+Launching the simulations is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` and ``lethe-fluid-particles`` executables are within your path, the simulations can be launched in parallel as follows:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles loading_particles.prm
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles settling_particles.prm
+
+The particle loading and settling simulation should look like this:
+
+.. raw:: html
+
+    <p align="center"><iframe width="560" height="315" src="https://www.youtube.com/embed/4uM51PCypZc?si=Xrisa4h87QLjvTWO" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-fluid-particles pneumatic-conveying.prm
+
+The pneumatic conveying simulation should look like this:
+
+.. raw:: html
+
+    <p align="center"><iframe width="560" height="315" src="https://www.youtube.com/embed/ESfSrmmlzYE?si=1RTsvFzcwvyelGme" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+.. note::
+   The pneumatic conveying simulation lasts 5 seconds in this example, but last 10 seconds in the video. You can change the end time in the parameter file.
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_.
```

Comment:
I feel is nice to have a preview of the example before reading the parameter files, This section is pretty big in this example. :)

Reply:
It's true that it is nice to have a preview like that before all the parameter file explanation... We should give some thought on maybe reformatting how we do our examples in general..

Reply:
I understand wanting to give a preview, but I don't like that it changes the order of the section. If someones wants to see the results they can go directly to the results section in the table of content.
Let's keep the order of the sections the same for uniformity. Otherwise eventually all examples will have different names. I feel like scrolling down for the results is pretty easy and intuitive.

Reply:
Oki dokie